models/Sketch/Sketch.js

```diff
@@ -75,6 +75,13 @@ class Sketch {
       });
   }
 
+  static addPreview(preview) {
+    if (!fs.existsSync(preview)) return;
```

Comment:
Should we add something to the console to let the user know there was an issue getting the file? Or throw an error?

Reply:
Maybe something like `console.warn('Error occurred while adding preview image, please check that file in place')`?

Reply:
Yea, what about 'Error occurred while adding preview image, please check the file path' ?

Reply:
Sounds good for me! Will update PR later today

Reply:
Done